package-lock.json

```diff
@@ -2504,7 +2504,6 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
-        "fsevents": "1.1.2",
```

Comment:
These changes don't belong in this PR.

Reply:
This is unintentional and caused by https://github.com/decidim/decidim/issues/1919. It's been almost 2 months now and the issue is not yet fixed in `npm` so I'm leaning towards going back to `yarn` since it does not suffer from this issue...